app/src/main/java/com/hamdan/forzenbook/data/repository/CreateAccountRepositoryImpl.kt

```diff
@@ -12,14 +12,21 @@ class CreateAccountRepositoryImpl(
         birthDay: String,
         email: String,
         location: String
-    ): Int {
-        return try {
-            service.createUser(firstName, lastName, birthDay, email, location)
-                .code()
+    ): Boolean {
+        try {
+            return when (
+                service.createUser(
+                    email, Date.valueOf(birthDay), firstName, lastName, location
+                ).code()
+            ) {
+                200 -> true
```

Comment:
`isSuccessful` takes the place of 200..299. What if you had a 201?

Reply:
isn't that fine in this situation? We just need to know if it succeeded, or is the only viable success code in this situation 200 because there could be a 202?

Reply:
oh I see what your saying, I should just place a isSuccesful instead of a specified 200